etc/launch-with-python-udf-server/cn.toml

```diff
@@ -0,0 +1,12 @@
+service-type = "CN"
+data-dir = "./mo-data"
+
+[log]
+level = "info"
+
+[cn]
+uuid = "dd1dccb4-4d3c-41f8-b482-5251dc7a41bf"
+port-base = 18000
+
+[cn.python-udf-client]
+server-address = "127.0.0.1:50051"
```

Comment:
Please consider using auto-generated port based on 'port-base' for internal communication ports

Reply:
This config is only for client. It is independent with the python server config. So should I change the both to 'port-base' or only change the server config (in file 'python-udf-server.toml')?

Reply:
Python UDF server address is configured outside the CN process and will not interact with HAKeeper (since it is design to have coupled lifecycle with CN), so the server-address configuration is required if python UDF server is not running on the default address (if any)